frontend/src/components/pod/Details.tsx

```diff
@@ -216,9 +276,14 @@ export function VolumeDetails(props: VolumeDetailsProps) {
   );
 }
 
-export default function PodDetails() {
+export interface PodDetailsProps {
+  showLogs?: boolean;
```

Comment:
Perhaps showLogs should be renamed to showLogsDefault, because that's what it is at the moment. If that prop is changed then the PodDetails component won't use that change.

Reply:
Got it. Will change.